java/src/jmri/jmrix/JmrixSystemsBundle_ca.properties

```diff
@@ -0,0 +1,64 @@
+# JmrixSystemsBundle.properties
+#
+# Revision $Revision$
+#
+# Default properties for the jmri.jmrix Systems menu
+#
+# Italian translation: Joan de Castro (joan276dca@yahoo.es) 16/08/2016
+#
+
+MenuTools = Sistema
+
+# Protocols that dont need their own file (yet)
+# share entries in this one
+
+MenuSystems = Sistema
+
+
+MenuItemAcela = Acela
+MenuItemBachrus = Bachrus
+MenuItemCAN = CAN
+MenuItemCBUS = CBUS
+MenuItemCMRI = CMRI
+MenuItemDirect = DirectDrive
+MenuItemEasyDCC = EasyDCC
+MenuItemEcos = ECoS
+MenuItemJMRIClient = JMRI Client di Rete
+MenuItemLocoNet = LocoNet
+MenuItemMRC = Model Rectifier Corp (MRC)
+MenuItemMaple = Maple Systems
+MenuItemNCE = NCE
+MenuItemOakTree = Oak Tree
+MenuItemOpenLCB = OpenLCB
+MenuItemSPROG = SPROG
+MenuItemTMCC = TMCC (Lionel)
+MenuItemSECSI = SECSI
+MenuItemSRCP = SRCP
+MenuItemWangrow = Wangrow
+MenuItemXPressNet = XPressNet
+MenuItemXPA = XPA
+MenuItemZimo = Zimo
+
+MenuItemCommandMonitor = Monitor de Comandes
+MenuItemTrafficMonitor = Monitor de Transit
+MenuItemSendCommand = Envia Comanda
+MenuItemSendFrame = Envio FMarc
+MenuItemFirmwareUpdate = Actualitza Firmware
+MenuItemSlotMonitor = Monitor de Slot
+MenuItemShowCabs = Visualizza Cabs
+MenuItemMacroCommand = Envia Macro
+MenuItemMacroEdit = Edita Macro
+MenuItemConsistEdit = Modifica Multi-Tracció
+MenuItemTrackPacketMonitor = Analitza Paquet DCC
+MenuItemClockMon = Monitor de Rellotge
+MenuItemConfigNodes = Configura Nodes
+MenuItemConsole = Consola
+MenuItemDiagnostics = Arrenca Diagnòstics
+MenuItemAssignments = Visualitza Assignacions
+MenuItemConfigTool = Configura les Connessions del Bus
+MenuItemEventTable = Taula d'Esdeveniments
```

Comment:
To put a single quote into a translation, you have to write 2 of these, like ``d''Esdeven...``
The (first) single quote is a so called escape character in properties files, treating the second as e text char.